radar-upload-frontend/src/App.vue

```diff
@@ -64,7 +64,9 @@ export default {
     async getAuth() {
       if (window.location.href.includes('login?code=')) {
         const authCode = window.location.search.replace('?code=', '');
-        const returnedToken = await getToken(authCode).catch(() => null);
+        // eslint-disable-next-line no-undef
+        const clientId = process.env.VUE_APP_CLIENT_ID || $VUE_APP_CLIENT_ID;
```

Comment:
So I like the suggestion of https://www.georg-ledermann.de/blog/2018/04/27/dockerize-and-configure-javascript-single-page-application/ to have a dedicated config js for this instead of spreading the config logic through all files. It makes the result much easier to troubleshoot. Note that there, they do (shorthand here):
```
class Config {
  static get CONFIG () {
    return {
      clientId: '$VUE_APP_CLIENT_ID'
    }
  }

  // When running in dev mode, replace the config with environment variables
  static value (name) {
    if (!(name in this.CONFIG)) {
      console.log(`Configuration: There is no key named "${name}"`)
      return
    }

    const value = this.CONFIG[name]

    if (!value) {
      console.log(`Configuration: Value for "${name}" is not defined`)
      return
    }

    if (value.startsWith('$VUE_APP_')) {
      // value was not replaced, it seems we are in development.
      // Remove $ and get current value from process.env
      const envName = value.substr(1)
      const envValue = process.env[envName]
      if (envValue) {
        return envValue
      } else {
        console.log(`Configuration: Environment variable "${envName}" is not defined`)
      }
    } else {
      // value was already replaced, it seems we are in production.
      return value
    }
  }
}
```

Alternatively, you could do something like
```js
export default let config = {
  clientId: process.env.VUE_APP_CLIENT_ID ||  '$VUE_APP_CLIENT_ID'
}
```

Reply:
ok will change.